diff --git a/databases/mysqlwsrep57-server/pkg-plist b/databases/mysqlwsrep57-server/pkg-plist
index 4e42ca872..412e1332b 100644
--- a/databases/mysqlwsrep57-server/pkg-plist
+++ b/databases/mysqlwsrep57-server/pkg-plist
@@ -164,6 +164,7 @@ share/man/man8/mysqld.8.gz
 %%DATADIR%%/swedish/errmsg.sys
 %%DATADIR%%/ukrainian/errmsg.sys
 %%DATADIR%%/uninstall_rewriter.sql
+%%DATADIR%%/wsrep_diag.sql
 %%DATADIR%%/wsrep_notify
 @sample(root,wheel,0644) %%ETCDIR%%/my.cnf.sample
 @dir(mysql,mysql,0750) %%ETCDIR%%/keyring
